lib/pure/options.nim

```diff
@@ -41,16 +38,12 @@
 ##
 ##    let found = "abc".find('c')
 ##    assert found.isSome and found.get() == 2
-##   
-## The ``get`` operation demonstrated above returns the underlying value, or
-## raises ``UnpackError`` if there is no value. Note that ``UnpackError`` inherits
-## from ``system.Defect``, and should therefore never be catched. Instead, rely on
-## checking if the option contains a value with ``isSome`` and ``isNone``.
 ##
-## There is another option for obtaining the value: ``unsafeGet``, but you must
-## only use it when you are absolutely sure the value is present (e.g. after
-## checking ``isSome``). If you do not care about the tiny overhead that ``get``
-## causes, you should simply never use ``unsafeGet``.
+## The ``get`` operation demonstrated above returns the underlying value, or
+## raises ``UnpackError`` if there is no value. Note that ``UnpackError``
+## inherits from ``system.Defect``, and should therefore never be catched.
```

Comment:
`catched` is very strange, maybe `caught`?

```suggestion
## inherits from ``system.Defect``, and should therefore never be caught.
```

Reply:
It was there from before :) Will change it.